src/Microsoft.Dnx.Compilation.CSharp/RoslynCompiler.cs

```diff
@@ -121,7 +121,10 @@ public class RoslynCompiler
             if (isPreprocessAspect)
             {
                 compilationSettings.CompilationOptions =
-                    compilationSettings.CompilationOptions.WithOutputKind(OutputKind.DynamicallyLinkedLibrary);
+                    compilationSettings.CompilationOptions
+                        .WithOutputKind(OutputKind.DynamicallyLinkedLibrary)
+                        .WithDelaySign(false)
+                        .WithCryptoKeyFile(null);
```

Comment:
Can you add some tests? The tests I [created](https://github.com/aspnet/dnx/blob/pakrym/reset-sign/test/Microsoft.Dnx.Compilation.CSharp.Tests/SigningFacts.cs) before (https://github.com/aspnet/dnx/blob/pakrym/reset-sign/test/Microsoft.Dnx.Compilation.CSharp.Tests/SigningFacts.cs) might be helpful.